gridstatus/caiso_utils.py

```diff
@@ -1,6 +1,21 @@
 import pandas as pd
 
 
+# NOTE: Currently the gridstatus.CAISO.default_timezone is in the caiso.py file, which imports caiso_utils.py and thus
+# it can't be set to the value without causing a circular import, which is why they are hardcoded here.
+def make_timestamp(time_str: str, today: pd.Timestamp, timezone: str = "US/Pacific"):
+    hour, minute = map(int, time_str.split(":"))
+    ts = pd.Timestamp(
+        year=today.year,
+        month=today.month,
+        day=today.day,
+        hour=hour,
+        minute=minute,
+    )
+    ts = ts.tz_localize(timezone, ambiguous=True)
+    return ts
+
+
 def check_latest_value_time(df: pd.DataFrame, column: str):
```

Comment:
It seems like the name of this function no longer represents what the function does. Something like `get_latest_time_value_on_today` is more accurate since it returns a timestamp from today.